src/java.base/share/classes/java/lang/reflect/code/op/CoreOp.java

```diff
@@ -3940,6 +3944,8 @@ public static VarOp var(Value init) {
 
     /**
      * Creates a var operation.
+     * <p>
+     * Null or empty string means unnamed variable
```

Comment:
I recommend updating all three methods to state:

    Creates a var operation modeling a variable, either a local variable or a parameter.

Then we can add to the second two methods:

    If the given name is {@code null} or an empty string then the variable is an unnamed variable.

We can then update the first method (that accepts no name) to:

    Creates a var operation modeling an unnamed variable, either an unnamed local variable or an unnamed parameter.